### PR TITLE
parser: Disable conditionally class constants

### PR DESCRIPTION
**Pull request by emilio**
Part of the fix for #1

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -152,6 +157,8 @@ pub struct BindgenOptions {
     pub enable_cxx_namespaces: bool,
     pub rename_types: bool,
     pub derive_debug: bool,
+    /// Werther to generate C++ class constants.
```
Comment (nox) [inline src/lib.rs]: Whether

**Comment by emilio** [comment]
Done

**Comment by emilio** [comment]
re-r? @nox

**Pull request merged**